shared/src/usecases/workflows/dockerAgent.ts

```
@@ -0,0 +1,167 @@
+    level: "info",
+    data: { containerId, name: params.containerName },
+  })
+
+  // 4) Create agent with simple tools
+  const tools: AgentTool[] = [
+    {
+      name: "emit_event",
+      description: "Emit a structured workflow event",
+      async execute(args: { type: string; data?: unknown }, ctx) {
+        await ctx.emit({
+          type: (args.type as any) ?? "status",
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)